keyboards/leeku/i2c.c

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Should not introduce its own i2c code, and use <https://docs.qmk.fm/#/i2c_driver>

Reply:
Replaced custom I2C driver with QMK I2C driver